public/scripts/project/metadata.js

```diff
@@ -62,6 +62,20 @@ define(function(require) {
     });
   }
 
+  // Look up the title for this project
+  function getTitle(root, callback) {
+    lock(function() {
+      getMetadata(root, function(err, value) {
```

Comment:
This is going to need to be fixed when you rebase to wrap the callback and not manually call unlock.